src/services/Frontline.js

```diff
@@ -124,6 +135,111 @@ class Frontline {
       }
     }
   }
+
+  /**
+   * Load Infinity
+   */
+
+  httpRequestWrapper = (method, URL) => {
+    return new Promise((resolve, reject) => {
+      const xhr_obj = new XMLHttpRequest();
+      xhr_obj.responseType = "json";
+      xhr_obj.open(method, URL);
+      xhr_obj.onload = () => {
+        const data = xhr_obj.response;
+        resolve(data);
+      };
+      xhr_obj.onerror = () => {
+        reject("failed");
+      };
+      xhr_obj.send();
+    });
+  };
+
+  async getProducts(apiEndpoint, params) {
+    return await this.httpRequestWrapper(
+      "GET",
+      `${apiEndpoint}/products.json?${params}`
+    );
+  }
+
+  footerHeight = () => {
+    var sum = 0;
+    LOAD_INFINITY_CLASSES.footer.split(",").forEach((className) => {
+      var element = document.querySelector(className);
+      if (typeof element != "undefined" && element != null) {
```

Comment:
Double-equality coerces both `undefined` and `null` into `false` so just using `element != null` suffices to check for `null or undefined`